CHANGELOG.md

```diff
@@ -6,6 +6,93 @@
+- Replace while preserving case
+
+### UI improvements
+
+Some new elements have been added or changed in the UI:
+
+- Rework the running kernels section
+- "Add cell" button at the notebook bottom
+- Dialog to display keyboard shortcuts as in the Classic Notebook (use <kbd>CTRL+SHIFT+H</kbd>)
+- Display first line of cell input and outputs when they are collapsed
+
+### Accessibility improvements
+
+JupyterLab is not yet fully accessible. Currently the focus is on making Notebook 7 accessible.
+A big part of the code is shared, though, and the following accessibility improvements are in JupyterLab 4:
+
+- Improved focus and keyboard navigation in the file browser
+- Add more roles and labels to UI elements
+- Main menu collapses to a hamburger menu if the web page is not wide enough to display all items.
+
+### Performance enhancements
+
+JupyterLab is now faster, thanks to the following improvements:
+
+- CSS rules optimization: CSS selectors have been optimized to improve web browser performance when many elements are present on a page.
+- Upgrade to CodeMirror 6: Especially for notebooks with many cells, the new CodeMirror version is far more efficient than the previous version. Large notebook should now load faster.
+- Notebook windowing: By rendering only the parts of a notebook that fit in the web browser viewport, JupyterLab is much more efficient. See an important note below.
+
+_Notebook windowing_ might add side effects with notebooks containing cells injecting new CSS styles or JavaScript. It may also be less convenient if some cell outputs are displaying [iframes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe). If you have issues with notebook rendering, try changing the user setting _Notebook_ > _Windowing mode_ to `defer` or `none`. (`none` should be used as a last resort, because it disables all optimizations.)
+
+### Real Time Collaboration
+
+JupyterLab 3.6 already made significant improvements to the Real Time Collaboration (RTC) feature.
+The feature has now been to a separate repository: [jupyter_collaboration](https://github.com/jupyterlab/jupyter_collaboration).
+The rationale is to limit the dependencies for users who don't need RTC. Separating RTC also helps organizations using JupyterLab that do not meet the specific requirements regarding file content management.
+
+To enable RTC, install the `jupyter-collaboration` package with:
+
+- with pip: `pip install "jupyter-collaboration>=1.0.0a0"`
+- with conda: _not yet available_
+
+Some highlights:
+
+- Support displaying multiple cursors and selections
+- Support registration of new shared model types
+
+### For developers
+
+Here are the main tool updates that may affect extension authors and developers:
+
+- TypeScript v5
+- Yarn v3
+- React v18
+- Lumino v2
+
+We also recommend NodeJS v18, the only version to remain supported in 2024 (see [Node release schedule](https://github.com/nodejs/release#release-schedule)).
```

Comment:
```suggestion
We recommend using Node.js v18 or newer, because older versions will reach end of life in 2023 or earlier (see [Node release schedule](https://github.com/nodejs/release#release-schedule)).
```

Technically, Node.js 20 (released earlier this month) will also remain supported in 2024.